lightbulb/commands/base.py

```diff
@@ -288,6 +302,18 @@ class CommandLike:
     """Whether this command will be enabled in DMs, if an application command."""
     app_command_bypass_author_permission_checks: bool = False
     """Whether invocations of this command will bypass author permission checks, if an application command."""
+    name_localizations: t.Mapping[t.Union[hikari.Locale, str], str] = dataclasses.field(default_factory=dict)
+    """
+    A set of name localizations for this command
+    
+    .. versionadded:: 2.3.0
+    """
+    description_localizations: t.Mapping[t.Union[hikari.Locale, str], str] = dataclasses.field(default_factory=dict)
+    """
+    A set of description localizations for this command
```

Comment:
Same as above. `"A mapping of locale to ... localizations for this command"` would be more appropriate